### ISSUE TITLE
Not able to create Case Observable

### ISSUE THREAD

**Issue by abhijeetasawant** [TheHive-Project/TheHive4py#162]
# Not able to create Case Observable

### Request Type
Bug 

### Work Environment

| Question              | Answer
|---------------------------|--------------------
| OS version (server)       | Ubuntu
| OS version (client)       | Ubuntu
| TheHive4py version / git hash   |  1.7.0.post1

### Problem Description
api.create_case_observable(case_id, obs) fails with error code 400 with error message : "Unknown attribute case_artifact.id.

Detailed response :
 {"tableName":"case_artifact","type":"AttributeCheckingError","errors":[{"name":"case_artifact.id","type":"UnknownAttributeError","message":"Unknown attribute case_artifact.id: {\"type\":\"JsonInputValue\",\"value\":null}","value":{"type":"JsonInputValue","value":null}}]}

### Steps to Reproduce
1. Create TheHiveApi.
2. Create CaseObservable. 
3. Call api.create_case_observable(case_id, obs)
4. returns error 400

### Possible Solutions
CaseObservable.jsonify() adds "id":null attribute to json. I think this attribute should be removed.

**Comment by Felix83000** [comment]
Same error here... Please Help :)
For now it is indeed impossible to create any observable...

**Comment by nadouani** [comment]
Hello @abhijeetasawant can you share a code sample that fails? I suppose that `obs` in `api.create_case_observable(case_id, obs)` is an instance of `thehive4py.models.CaseObservable` ?

**Comment by abhijeetasawant** [comment]
Yes obs is an instance of thehive4py.models.CaseObservable ?      
 Sample Code : 

        api = TheHiveApi(url, api_key)
        obs = CaseObservable(dataType=data_type,
                             message=description,
                             data=data)

        result = api.create_case_observable(case_id, obs)

**Comment by abhijeetasawant** [comment]
Yes. Problem is solved. Thank you.

**Comment by Felix83000** [comment]
Ok for me. Thank you
